doc/release-notes/5.12-release-notes.md

```diff
@@ -0,0 +1,171 @@
+
+Please note that an upgrade to Payara 5.2021.8 or higher is required to make use of this. Payara 5.2021.5 threw exceptions, as explained in PR #8823.
+
+## Major Use Cases and Infrastructure Enhancements
+
+Changes and fixes in this release include:
+
+- Administrators can configure an S3 store used in Dataverse to support users uploading/downloading files via Globus File Transfer. (PR #8891)
+- Administrators can configure a RemoteOverlay store to allow files that remain hosted by a remote system to be added to a dataset. (PR #7325)
+- Administrators can configure the Dataverse software to send archival Bag copies of published dataset versions to any S3-compatible service. (PR #8751)
+- Users can see information about a dataset's parent collection(s) in the OAI-ORE metadata export. (PR #8770)
+- Users and administrators can now use the OAI-ORE metadata export to retrieve and assess the fixity of the original file (for ingested tabular files) via the included checksum. (PR #8901)
+- Archiving via RDA-conformant Bags is more robust and is more configurable. (PR #8773, #8747, #8699, #8609, #8606, #8610)
+- Users and administrators can see the archival status of the versions of the datasets they manage in the dataset page version table. (PR #8748, #8696)
+- Administrators can configure messaging between their Dataverse installation and other repositories that may hold related resources or services interested in activity within that installation. (PR #8775)
+- Collection managers can create templates that include custom instructions on how to fill out specific metadata fields.
+- Dataset update API users are given more information when the dataset they are updating is out of compliance with Terms of Access requirements (Issue #8859) 
+- Adds a new setting (:ControlledVocabularyCustomJavaScript) that allows a JavaScript file to be loaded into the dataset page for the purpose of showing controlled vocabulary as a list (Issue #8722)
+- Fixes an issue with the Redetect File Type API (Issue #7527)
+- Updates the Import Dataset DDI API to allow the import of Terms of Use (Issue #8715)
+- Optimizes some code to improve application memory usage (Issue #8871)
+- Fixes sample data to reflect custom licenses.
+- Fixes the Archival Status Input API (available to superusers) (Issue #8924)
+- Small bugs have been fixed in the dataset export in the JSON and DDI formats; eliminating the export of "undefined" as a metadata language in the former, and a duplicate keyword tag in the latter. (Issue #8868)
+
+## New DB Settings
+The following DB settings have been added:
+- `:ShibAffiliationOrder`  - Select the first or last entry in an Affiliation array
+- `:ShibAffiliationSeparator` (default: ";") - Set the separator for the Affiliation array
```

Comment:
This is missing the additions in #8894 and it looks like other sections in that PR haven't been copied in.

Reply:
I think I've incorporated everything from Data Commons. Please take another look. And feel free to edit the notes if there's anything missing.

Reply:
So what's the final word on this, can this be marked as resolved? To me it also looks like all the new database options from #8894 are here, and so are the upgrade instructions etc. But it would be easy for me to miss something since it wasn't my PR.

Reply:
Looks good.